example/src/components/RoomParticipant.tsx

```diff
@@ -18,23 +18,32 @@ import { PinButton } from './buttons/PinButton';
 
 type RoomParticipantProps = {
   participant: Membrane.Participant;
+  trackId?: string;
   onPinButtonPressed?: (string) => void;
   focused?: boolean;
   pinButtonHiddden?: boolean;
   tileSmall?: boolean;
 };
 
 export const RoomParticipant = ({
-  participant: { metadata, tracks, type, id },
+  participant,
+  trackId = '',
```

Comment:
why is it empty string by default? empty track id doesn't make sense, use nulls if there is no video track

Reply:
Changed to null.